Sources/RealDeviceMapLib/Webhook/WebHookRequestHandler.swift

```diff
@@ -1230,9 +1257,11 @@ public class WebHookRequestHandler {
                     response.respondWithError(status: .notFound)
                     return
                 }
-                device.accountUsername = nil
-                try device.save(mysql: mysql, oldUUID: device.uuid)
-                response.respondWithOk()
+                if (rpc12Count[device.accountUsername!] ?? 0) == 0 {
```

Comment:
Imo that's not a good change.that will lock the device on same account if "logged_out" is sended...
I assume that's your workaround because GC logs out?

Reply:
Yes, this is the workaround. This is the only/best way I could make it "remember" the device.accountUsername so that when it goes to get_account after GC tells it to log out, it will log back into the same account. Without this, accounts assigned to a leveling instance would be in cooldown, so get_account would not select that account. This is why accounts struggle to reach 30 with GC versions that report rpc12.
I have not seen any issues with this in testing. I have never seen an account that gets into type == "logged_out"  and has rpc12Count[device.accountUsername!] > 0 except the ones that we want to keep using until they reach maxRpc12. Because an encounter resets rpc12Count, it would be an edge case when this happen (at least for leveling and circle instances). I have not seen rpc12s for quests, so can't comment if this bit of code could cause a problem. But, at worst, it would continue to log into that account and accumulate rpc12 until it hit the limit and disabled the account, thus, the account wouldn't be picked because its no longer valid.

Reply:
AS GC is fixing that behavior in next release, this workaround in logged_out is a blocker for merging, pls remove it. You can still run the actual commit :)